### ISSUE TITLE
background color differs between st and termite

### ISSUE THREAD

**Issue by ossi-frage** [dylanaraps/pywal#465]
I've been noticing a difference in the background colours between termite and st. This happens when using the default `colors-wal-st.h` as well as with a custom template in which "colour0" is actually set to `{colour0}` rather than the default `{background}`, and when the following is set thus

```
unsigned int defaultbg = 256
/* [rather than] unsigned int defaultbg = 0 */
``` 

Another thing I've observed is that this happens when "colour0" differs from the specified "background" colour, wherein termite gets the correct background colour while st gets `color0` as the background. In this case, termite has the correct background that is `#000000` while st has a background `#0a1124` (which is `color0`)

## Screenshots 
(apologies if the screenshots are too large)
 
st
![pywal-background-disparity_st](https://user-images.githubusercontent.com/32803930/68479719-048e4900-0259-11ea-99aa-f0b705d95ec0.png)

termite
![pywal-background-disparity_termite](https://user-images.githubusercontent.com/32803930/68479720-048e4900-0259-11ea-9c1a-5cb6daf06dc9.png)

**Comment by dylanaraps** [comment]
Does changing `defaultbg` fix the issue in its entirety?

**Comment by ictinaetus** [comment]
Nope. I think I mentioned this in the description (albeit not as clearly I think!)

**Comment by dylanaraps** [comment]
I personally use `pywal` *without* the `.h` file (with `st`). I just add the `cat` command to my `.shellrc` and all is well. Have you tried it this way?

**Comment by ictinaetus** [comment]
Will try this out. Sounds a lot more convenient that recompiling `st` each time!

**Comment by ictinaetus** [comment]
I thought I'd let you know that I'm having issues with this too! Turns out that I had the `cat` command in my `shellrc` all along (not sure why I didn't remember having added it). Tried another theme that I got off someone on reddit and this happened with st:
![pywal-background-disparity_st_2](https://user-images.githubusercontent.com/32803930/68539930-74fbae00-03b0-11ea-851e-23070af94dea.png)

This is a similar theme wherein `color0` actually is whiteish for some reason. It seems `pywal` sets the background correctly for `termite` but not for `st`. Also not sure why I get that odd bar of colour on the top in `st`. It disappears if a new window is spawned.
